docs/metrics/labels.md

```diff
@@ -31,6 +31,7 @@ Currently we support this for:
 
 - Azure Service Bus
 - Azure Storage Queue
+- Azure SQL Database
```

Comment:
Nitpick - Can we alphabetically order them?

Sorry about that :)

Reply:
haha - no problem. I can't believe I went to the bother of alphabetizing the index.md page, but not that.

Reply:
No worries :D